src/modules/navigation/hooks/helpers.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  isShortcut,
+  isNote,
+  shouldBeOpenedByOnlyOffice,
+  isDirectory
+} from 'cozy-client/dist/models/file'
+
+import type { File } from 'components/FolderPicker/types'
+import { TRASH_DIR_ID } from 'constants/config'
+import { isNextcloudShortcut } from 'modules/nextcloud/helpers'
+import { makeOnlyOfficeFileRoute } from 'modules/views/OnlyOffice/helpers'
+
+interface ComputeFileTypeOptions {
+  isOfficeEnabled?: boolean
+}
+
+interface ComputePathOptions {
+  type: string
+  pathname: string
+  isPublic: boolean
+}
+
+export const computeFileType = (
```

Comment:
@paultranvan I'd like to take this opportunity to point you to this function where you'll find a good number of paths to files on drive

Reply:
Nice! This might be worth moving it to cozy-client file's models, as we might have the use elsewhere